test/notifications/notification_server_test.exs

```diff
@@ -613,4 +613,55 @@ defmodule Notifications.NotificationServerTest do
       refute String.contains?(log, "new_notification")
     end
   end
+
+  # rename to activated_detour?
```

Comment:
I don't mind the current naming as-is. If this is a comment for some future, fine to leave it, but seems like this can be removed.

Reply:
Oh yes, thank you!
This used to be named `detour/3` and I changed it to `detour_activated/2` and didn't notice this comment was left over

Reply:
https://github.com/mbta/skate/compare/76a38d89c7a3879197589be193fb8d65d5e28cb4..f6923945f764b44ad4a0da4d65097f226e087e25